src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
   pageTitle="Tabs"
 >
   <sky-docs-demo-page-summary>
-    Sample description.
+    Tabs are containers for dividing large amounts of content on a page.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include counts"
+          propertyName="includeCounts"
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include add and close"
+          propertyName="includeAddAndClose"
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <sky-tabset *ngIf="!demoSettings.includeAddAndClose">
+      <sky-tab *ngFor="let tab of tabArray; let i = index"
+        [tabHeaderCount]="demoSettings.showCounts ? tab.tabHeaderCount : null"
+        [tabHeading]="tab.tabHeading"
+      >
+        {{ tab.tabContent }}
+      </sky-tab>
+    </sky-tabset>
+
+    <sky-tabset *ngIf="demoSettings.includeAddAndClose"
+      (newTab)="onNewTabClick()"
+    >
+      <sky-tab *ngFor="let tab of tabArray; let i = index"
+        [tabHeaderCount]="demoSettings.showCounts ? tab.tabHeaderCount : null"
+        [tabHeading]="tab.tabHeading"
+        (close)="onCloseClick(i)"
+      >
+        {{ tab.tabContent }}
+      </sky-tab>
+    </sky-tabset>
+  </sky-docs-demo>
+
+  <sky-docs-code-examples>
+    <sky-docs-code-example
+      heading="Static tabs"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/static"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Dynamic tabs bound to an array"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/dynamic"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Static tabs with counts"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/counts"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Static tabs with add and close"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/static-add-close"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Dynamic tabs with add and close"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/dynamic-add-close"
+    ></sky-docs-code-example>
+  </sky-docs-code-examples>
+
   <sky-docs-design-guidelines>
-    See documentation for how to setup design guidelines.
+    
+      <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+        >
```

Comment:
I would double-check the tab characters in this file.

Reply:
I cleaned up the tab characters and other spacing. I'm resolving this comment, but let me know if I missed anything.